src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/SpanUtility.cs

```diff
@@ -24,236 +24,22 @@ public static bool IsValidIndex<T>(Span<T> span, int index)
-        /// </summary>
-        /// <remarks>
-        /// Parameters are intended to be constant values.
-        /// </remarks>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool TryWriteChars(Span<char> span, char a, char b, char c, char d, char e)
-        {
-            if (span.Length >= 5)
-            {
-                ulong abcd64;
-                if (BitConverter.IsLittleEndian)
-                {
-                    abcd64 = ((ulong)d << 48) | ((ulong)c << 32) | ((ulong)b << 16) | a;
-                }
-                else
-                {
-                    abcd64 = ((ulong)a << 48) | ((ulong)b << 32) | ((ulong)c << 16) | d;
-                }
-                ref char rDest = ref MemoryMarshal.GetReference(span);
-                Unsafe.WriteUnaligned<ulong>(ref Unsafe.As<char, byte>(ref rDest), abcd64);
-                Unsafe.Add(ref rDest, 4) = e;
-                return true;
-            }
-            else
-            {
-                return false;
-            }
-        }
-
-        /// <summary>
-        /// Tries writing six chars to the span. If success, returns true. If the span is not large
-        /// enough to hold six chars, leaves the span unchanged and returns false.
-        /// </summary>
-        /// <remarks>
-        /// Parameters are intended to be constant values.
-        /// </remarks>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool TryWriteChars(Span<char> span, char a, char b, char c, char d, char e, char f)
-        {
-            if (span.Length >= 6)
-            {
-                ulong abcd64;
-                uint ef32;
-                if (BitConverter.IsLittleEndian)
-                {
-                    abcd64 = ((ulong)d << 48) | ((ulong)c << 32) | ((ulong)b << 16) | a;
-                    ef32 = ((uint)f << 16) | e;
-                }
-                else
-                {
-                    abcd64 = ((ulong)a << 48) | ((ulong)b << 32) | ((ulong)c << 16) | d;
-                    ef32 = ((uint)e << 16) | f;
-                }
-                ref byte rDest = ref Unsafe.As<char, byte>(ref MemoryMarshal.GetReference(span));
-                Unsafe.WriteUnaligned<ulong>(ref rDest, abcd64);
-                Unsafe.WriteUnaligned<uint>(ref Unsafe.AddByteOffset(ref rDest, (nint)sizeof(ulong)), ef32);
-                return true;
-            }
-            else
-            {
-                return false;
-            }
-        }
-
         /// <summary>
         /// Tries writing a 64-bit value as little endian to the span. If success, returns true. If
         /// the span is not large enough to hold 8 bytes, leaves the span unchanged and returns false.
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool TryWriteUInt64LittleEndian(Span<byte> span, int offset, ulong value)
```

Comment:
Remove this and just use `BinaryPrimitives.TryWriteUInt64LittleEndian`?

Reply:
thanks!